test/functional/interface_bitcoin_cli.py

```diff
@@ -28,7 +28,10 @@
 BLOCKS_VALUE_OF_ZERO = 'error: the first argument (number of blocks to generate, default: 1) must be an integer value greater than zero'
 TOO_MANY_ARGS = 'error: too many arguments (maximum 2 for nblocks and maxtries)'
 WALLET_NOT_LOADED = 'Requested wallet does not exist or is not loaded'
-WALLET_NOT_SPECIFIED = 'Wallet file not specified'
+WALLET_NOT_SPECIFIED = 'For the CLI, specify the "-rpcwallet=<filename>" option before the command'
```

Comment:
Again, I don't think "For the CLI" message is useful if it doesn't have a "as opposed to the ".

Reply:
I've updated the error message as I've commented it [here](https://github.com/bitcoin/bitcoin/pull/26990#discussion_r1131669863).